web/themes/custom/pineapple/sass/base/_base.scss

```diff
@@ -16,6 +16,7 @@ body {
   display: block;
   position: relative;
   padding-bottom: 65px;
+  margin: 0;
```

Comment:
What's the reason for this?

Reply:
You told me to do this without reasoning about a month ago and I forgot until now. I cannot remember the purpose, sorry.